src/libslic3r/ExtrusionEntity.hpp

```diff
@@ -120,6 +126,11 @@ inline bool is_solid_infill(ExtrusionRole role)
         || role == erIroning;
 }
 
+inline bool is_top_solid_infill(ExtrusionRole role)
+{
+    return role == erTopSolidInfill;
+}
```

Comment:
I think you can add erIroning here.
Also, why add it if you don't use it?

Reply:
I will remove those functions again, they are not used anymore